src/pybind/mgr/dashboard/controllers/service.py

```diff
@@ -32,7 +32,7 @@ def known_types(self) -> List[str]:
     @raise_if_no_orchestrator([OrchFeature.SERVICE_LIST])
     def list(self, service_name: Optional[str] = None) -> List[dict]:
         orch = OrchClient.instance()
-        return [service.to_json() for service in orch.services.list(service_name=service_name)]
+        return [service.to_json(True) for service in orch.services.list(service_name=service_name)]
```

Comment:
And please, in this very case, please let's add the kwarg explicitly, as reading `to_json(True)` and not getting any JSON string at all might become a brain-wrecker when your future you has to debug this:
```suggestion
        return [service.to_json(as_dict=True) for service in orch.services.list(service_name=service_name)]
```